build.gradle.kts

```diff
@@ -66,7 +66,7 @@ val schemaUrl = "https://opentelemetry.io/schemas/$semanticConventionsVersion"
 
 val downloadSemanticConventions by tasks.registering(Download::class) {
   src(semanticConventionsRepoZip)
-  dest("$buildDir/semantic-conventions-${semanticConventionsVersion}/semantic-conventions.zip")
+  dest("${layout.buildDirectory.asFile.get()}/semantic-conventions-${semanticConventionsVersion}/semantic-conventions.zip")
```

Comment:
from what I understood the intended usage would be `dest(layout.buildDirectory.file("semantic-conventions-${semanticConventionsVersion}/semantic-conventions.zip"))` (didn't test) to allow for lazy resolution of the build dir

Reply:
Will take a look. Thanks for the advice!

Reply:
Had to drop the leading `/` but works otherwise